api/vendor/github.com/openshift/assisted-service/models/versioned_host_requirements.go

```diff
@@ -35,6 +38,10 @@ type VersionedHostRequirements struct {
 func (m *VersionedHostRequirements) Validate(formats strfmt.Registry) error {
 	var res []error
 
+	if err := m.validateDPUWorkerRequirements(formats); err != nil {
```

Comment:
it can pass this validation but will fail in validateWorkerRequirements.  I think you should not run validateWorkerRequirements if it's DPU worker

Reply:
Should not fail. Tested it locally.  Will test more in anycase